addons/account/views/account_move_views.xml

```diff
@@ -705,18 +705,18 @@
                                                attrs="{
                                                     'required': [('display_type', 'not in', ('line_section', 'line_note'))],
                                                     'invisible': [('display_type', 'in', ('line_section', 'line_note'))],
-                                               }"/>
+                                               }"  optional="show"/>
                                         <field name="partner_id"
                                                domain="['|', ('parent_id', '=', False), ('is_company', '=', True)]"
                                                attrs="{'column_invisible': [('parent.type', '!=', 'entry')]}"/>
-                                        <field name="name" widget="section_and_note_text"/>
+                                        <field name="name" widget="section_and_note_text"  optional="show"/>
```

Comment:
It seems weird to remove the field `name` and I don't see anywhere in the specs what specific fields should be optional. Has it been verified with a PO?

Reply:
nope, there isn't anything mentioned what fields to hide, but now I have asked for AVW's opinion in task.